src/app/home/home.component.ts

```diff
@@ -45,24 +59,31 @@ export class HomeComponent extends WithUnsubscribe() implements OnInit {
           }),
         );
       });
-  }
 
-  public openAppNav() {
-    this.store.dispatch(new layoutStore.actions.OpenAppNav());
+    // When router changes and sideNavMode is 'over', hide the sideNav.
+    this.currentSubroute$.subscribe(() => {
+      if (this.sideNavMode === 'over' && this.sideNavOpen === true) {
```

Comment:
Please, fix lint error: `ERROR: /src/app/home/home.component.ts[65, 42]: This expression is unnecessarily compared to a boolean. Just use it directly.`
Also, always run lint/test/e2e before commit.

Reply:
@huadream Hello, we are ready to give your PR to our QA, please fix lint error described above

Reply:
I am very sorry that I get out for several days and e2e test didn't fix.